src/utils/general.ts

```diff
@@ -176,7 +176,7 @@ export function getQueryForDepositsAndRewards(accountid: string) {
 
 export function convertFromUtezToTez(amountInUtez: number) {
   const tezAmount = amountInUtez / 1000000;
-  return `${tezAmount} XTZ`;
+  return tezAmount;
```

Comment:
This is not correct. The change should be applied only to Gas and Storage fields. For example: block.consumed_gas, operation.gas_limit, operation.storage_limit.